docs/source/index.rst

```diff
@@ -1,118 +1,136 @@
+.. code-block:: console
 
-The easiest way to get started is to install the latest version of all Dissect projects from
-`PyPI <https://pypi.python.org/pypi/dissect>`_:
+    $ target-query pc.img -f users
+    
+    <windows/user ... name='systemprofile' ...>
+    <windows/user ... name='LocalService' ...>
+    <windows/user ... name='NetworkService' ...>
+    <windows/user ... name='Mr. Evil' ...>
+    
+To see what other useful artefacts you can query use ``-l``:
 
 .. code-block:: console
 
-    $ pip install dissect
+    $ target-query pc.img -l
 
-It is recommended that you use a `virtual environment <https://docs.python.org/3/tutorial/venv.html>`_.
+This will list all Dissect functions that are available for this image.
+For a complete list of all functions, regardless of the image, just omit the image argument.
 
-.. note ::
-    .. include:: /versions.rst
 
-To quickly get familiar with Dissect, read on to :doc:`/usage/introduction`. If you're interested in what makes it
-tick, continue reading at :doc:`/overview/index`.
+Key features
+------------
+
+Dissect is a powerful artefact extractor and parser that saves you a lot of time.
+With Dissect you can:
 
-Installing individual projects
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* Quickly **extract** artefacts from any source (like IMG, EWF, Kape, DD, VDI, PVM)
+* Access artefacts from almost any **OS** (like Windows, macOS, Linux/Unix, ESXi)
+* Access almost any **filesystem** (like NTFS, EXT, XFS, QNX6)
+* **Parse** (binary) logs, registry entries, cookies, history and more, directly from the source
+* **Export** findings to text, JSON, CSV or stream to datastores like Splunk, Elastic, MySQL
+* **Feed, filter and script** to process found data in any way you like
+* Investigate images with powerful tools like **target-shell**, **target-mount** or the **Python API**
 
-Dissect is a collection of libraries and tools, so every Dissect project is individually installable. You'll find that
-installing the ``dissect`` package from PyPI will install all available ``dissect.*`` packages for you! If, however,
-you just want to work with the NTFS and registry parsers in your own scripts, you can opt to only install or depend
-on those specific projects:
+Using these features, you can for instance (among other things):
+
+* Make a timeline of the event log from all investigation data at once
+* Identify anomalies in artefacts such as services, tasks and run keys over a large amount of investigation data
+* Perform incident response on partially ransomwared (encrypted) virtual machine disks
+* Perform complex IOC checks on thousands of hosts in a couple of hours
+* Export all USN journal records from a Bitlocker-encrypted disk to Splunk without waiting for decryption
+* Collect forensic artefacts from all live virtual machines directly from the hypervisor with no down time
+* Export all artefacts easily to any data format you want, for example CSV, JSON or Avro, or stream directly to Splunk or Elastic
```

Comment:
```suggestion
Using these features, you can for instance (among other things):

* Make a **timeline** of the event log from all your investigation data at once.
* Identify **anomalies** in artefacts such as services, tasks and run keys over a large amount of investigation data.
* Do incident response on partially **ransomwared** (encrypted) virtual machine disks.
* Extract artefacts from **LVM, RAID** and other disk setups, without any manual configuration.
* Perform complex **IOC checks** on thousands of hosts in a couple of hours.
* Collect forensic artefacts from all **live** virtual machines directly from the hypervisor with no down time.
* Export all artefacts easily to any data format you want, for example CSV, JSON or Avro, or stream directly to Splunk or Elastic.
```
Removed this line since this feature is not yet in the public version.
>* Export all USN journal records from a Bitlocker-encrypted disk to Splunk without waiting for decryption.